idd/Energy+.idd.in

```diff
@@ -50272,6 +50272,236 @@ ZoneHVAC:EquipmentConnections,
+       \key ZoneHVAC:UnitHeater
+       \key ZoneHVAC:UnitVentilator
+       \key ZoneHVAC:VentilatedSlab
+       \key ZoneHVAC:WaterToAirHeatPump
+       \key ZoneHVAC:WindowAirConditioner
+       \key ZoneHVAC:Baseboard:RadiantConvective:Electric
+       \key ZoneHVAC:Baseboard:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:RadiantConvective:Steam
+       \key ZoneHVAC:CoolingPanel:RadiantConvective:Water
+       \key ZoneHVAC:Baseboard:Convective:Electric
+       \key ZoneHVAC:Baseboard:Convective:Water
+       \key ZoneHVAC:HighTemperatureRadiant
+       \key ZoneHVAC:LowTemperatureRadiant:VariableFlow
+       \key ZoneHVAC:LowTemperatureRadiant:ConstantFlow
+       \key ZoneHVAC:LowTemperatureRadiant:Electric
+       \key ZoneHVAC:Dehumidifier:DX
+       \key ZoneHVAC:IdealLoadsAirSystem
+       \key ZoneHVAC:RefrigerationChillerSet
+       \key WaterHeater:HeatPump:PumpedCondenser
+       \key WaterHeater:HeatPump:WrappedCondenser
+       \key HeatExchanger:AirToAir:FlatPlate
+       \key AirLoopHVAC:UnitarySystem
+   A4, \field Zone Equipment Name
+       \required-field
+       \type object-list
+       \object-list ZoneEquipmentNames
+   A5, \field Zone Equipment Outlet Node Name
+       \note Only used for airflow equipment.
+       \note The outlet node from the zone equipment that will be split to the spaces.
+       \type node
+   A6, \field Thermostat Control Method
+       \note SingleSpace satisfies the thermostat in the Control Space Name
+       \note Maximum satisfies the thermostat in the connected spaces with the highest deviation from setpoint
+       \note Ideal ignores the Space Output Fractions and distributes the output to match the current space sensible loads.
+       \type choice
+       \key SingleSpace
+       \key Maximum
+       \key Ideal
+       \default SingleSpace
+   A7, \field Control Space Name
+       \note Used with SingleSpace thermostat control method
+       \type object-list
+       \object-list SpaceNames
+   A8, \field Space Fraction Method
+       \note The basis used to autosize the space output fractions.
+       \type choice
+       \default DesignCoolingLoad
+       \key DesignCoolingLoad
+       \key DesignHeatingLoad
+       \key FloorArea
+       \key Volume
+       \key PerimeterLength
+   A9, \field Space 1 Name
+       \begin-extensible
+       \required-field
+       \type object-list
+       \object-list SpaceNames
+   N1, \field Space 1 Fraction
+       \note Fraction of this zone equipment output or airflow distributed to this space.
+       \default autosize
+       \units dimensionless
+       \minimum 0.0
+       \maximum 1.0
+       \autosizable
+  A10, \field Space 1 Supply Node Name
+       \note Only used for airflow equipment
+       \note Matches a SpaceHVAC:EquipmentConnections Inlet Node Name
+       \type node
+  A11, \field Space 2 Name
+       \required-field
+       \type object-list
+       \object-list SpaceNames
```

Comment:
And I see that Space 2 Name is a required-field, as is Space 3 Name. I think that's a mistake. There can technically be only 1 space in a zone so requiring any fields after Space 1 might be troublesome?

Reply:
Required is a mistake. If you want to make these idd changes please go ahead.

Reply:
I will make the few changes to the idd. (notes, required-field, min-fields)